filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest_test.go

```diff
@@ -137,23 +160,106 @@ func TestAuthorizeRequestFilter(t *testing.T) {
 		},
 		{
 			msg:               "Wrong Query Data Type",
+			filterName:        "opaAuthorizeRequest",
 			bundleName:        "somebundle.tar.gz",
 			regoQuery:         "envoy/authz/allow_object_invalid_headers",
 			requestPath:       "/allow",
+			requestMethod:     "GET",
 			contextExtensions: "",
 			expectedStatus:    http.StatusInternalServerError,
 			expectedBody:      "",
 			expectedHeaders:   make(http.Header),
 			backendHeaders:    make(http.Header),
 			removeHeaders:     make(http.Header),
 		},
+		{
+			msg:             "Allow With Body",
```

Comment:
what about tests that exceeds the max body size?

Reply:
@szuecs Wdyt, is that test enough or should we replicate it here as well?

Reply:
ok thanks!